dpctl/tests/test_sycl_device.py

```diff
@@ -552,6 +557,8 @@ def check_profiling_timer_resolution(device):
     check_repr,
     check_get_global_mem_size,
     check_get_local_mem_size,
+    check_profiling_timer_resolution,
```

Comment:
unrelated?

Reply:
Yes, I realized this check added earlier was not exercised, so added it along with the `check_platform`